java/widget-store/src/main/java/com/example/widgetstore/config/DBOSLifecycle.java

```diff
@@ -0,0 +1,34 @@
+package com.example.widgetstore.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.ContextClosedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+import dev.dbos.transact.DBOS;
+
+@Component
+public class DBOSLifecycle  {
```

Comment:
Looks good. But do users need to remember to add this for every Spring app?

Reply:
for now, yes. Eventually, I want to have a dbos.spring JAR that includes stuff like this so they just have to implement @Bean functions for DBOSConfig and the code that registers workflows and steps and the library does the rest.